tests/core/test_trackers.py

```diff
@@ -567,3 +568,13 @@ def test_last_executed_has_not_name():
     tracker = get_tracker(events)
 
     assert tracker.last_executed_action_has("another") is False
+
+
+@pytest.mark.parametrize("key", ["asfa", "htb", "2"])
+def test_tracker_without_slots(key):
+    tracker = DialogueStateTracker.from_dict("any", [])
+    assert key in tracker.slots
```

Comment:
imho this tests the solution rather than the causing problem.
Can we write a test which does sth like

```python
event = SlotSet(...)
tracker = DialogueStateTracker.from_dict("any", [])
event.apply_to(tracker)

assert that no log
assert that slot is set
```

Reply:
I kept the `key in tracker.slots` as this is the way we use it in rasa also and replaced the other part with a `tracket.get_slot` which should log an INFO message if the slot does not exist